scripts/route-pre-down.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# redirect stdout/stderr to a file
+#exec &>>route-pre-down.log
+
+#Print Date
+NOW=$(date +"%Y-%m-%d %T")
+
+echo "${NOW}: route-pre-down script: Start "
+
+echo "Sending exit signal to transmission."
+
+transmission-remote --exit &
```

Comment:
This will fail if authentication is needed. You can reproduce the following steps to make it handle transmission being protected:

https://github.com/haugene/docker-transmission-openvpn/blob/93d0141d928ec661411e2c6f0e079aeff2c018dd/openvpn/pia/update-port.sh#L10-L13
https://github.com/haugene/docker-transmission-openvpn/blob/93d0141d928ec661411e2c6f0e079aeff2c018dd/openvpn/pia/update-port.sh#L100

Then `transmission-remote $myauth --exit`

Reply:
Thanks for pointing this out, I will make the change later.

Reply:
thinking to update with this change soon?